boards/atxmega-a1u-xpro/include/periph_conf.h

```diff
@@ -29,6 +29,8 @@ extern "C" {
 
 #include "periph_cpu.h"
 
+#define AVR8_NESTED_IRQ   (1)
```

Comment:
Is this really a property of the board?
I think this should be defined by the CPU.

Reply:
moved to Kconfig